.github/workflows/deploy.yml

```
@@ -1,3 +1,4 @@
+name: CI/CD – Build & Publish Docker Images
 name: CI/CD – Build & Publish Docker Image
```

actionlint (1.7.7)

key "name" is duplicated in workflow. previously defined at line:1,col:1

(syntax-check)